apps/api/src/Command/GreencheckCsvCheckerCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace App\Command;
+use Doctrine\ORM\EntityManagerInterface;
+use Enqueue\Client\ProducerInterface;
+use Enqueue\Client\Message;
+use Enqueue\Client\MessagePriority;
+use Enqueue\Util\JSON;
+use Predis\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use TGWF\Greencheck\Sitecheck;
+/**
+ * GreencheckCsvCheckerCommand.
+ *
+ * Grabs all domains from a csv file in the first column and runs them through the queue
+ */
+class GreencheckCsvCheckerCommand extends Command
+{
+    /**
+     * @var ProducerInterface
+     */
+    private $producer;
+    public function __construct(
+        $name = null,
+        ProducerInterface $producer
+    ) {
+        parent::__construct($name);
+        $this->producer = $producer;
+    }
+    /**
+     * Configure the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('tgwf:greencheck:csvchecker')
+            ->setDescription('Checks all domains in a csv file')
+            ;
+    }
+    /**
+     * Execute the command.
+     *
+     * @param InputInterface  $input  Input
+     * @param OutputInterface $output Output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+
+        if (($handle = fopen("tgwf-check.csv", "r")) !== FALSE) {
```

Comment:
we could improve this with a InputArgument on the configure() above so you can define which file to import instead of hardcoding it. Might make the python script easier?

Reply:
Good point. I've created #27 for this, as I'd like to make sure we don't miss it.